packages/paste-design-tokens/tokens/aliases/font-family.yml

```diff
@@ -1,3 +1,7 @@
 aliases:
   font-family-10: "'Inter var experimental', 'Inter var', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif"
   font-family-20: "'Fira Mono', 'Courier New', Courier, monospace"
+  font-family-ja: "'Inter var experimental', 'Inter var', Hiragino Sans, 'ヒラギノ角ゴ ProN W3', 'Hiragino Kaku Gothic ProN', 'メイリオ', Meiryo, Osaka, 'MS PGothic', sans-serif"
```

Comment:
Question: the other font family aliases use a number, but it felt kind of weird for this case. What should the alias name be?

`font-family-ja`, `font-family-30`, or `font-family-ja-10`?

Reply:
General naming convention for alias is that they are number steps. They are generic, buckets of values that have no meaning, and we don't need to think about naming them when adding more.

I'd go with ont-family-30 personally